scipy/stats/_stats_py.py

```diff
@@ -6996,19 +6996,18 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
     nobs2 : array_like
         The number(s) of observations of sample 2.
     equal_var : bool, optional
-        If True (default), perform a standard independent 2 sample test
-        that assumes equal population variances [1]_.
-        If False, perform Welch's t-test, which does not assume equal
-        population variance [2]_.
+        If True (default), perform a standard independent 2 sample test that
+        assumes equal population variances [1]_. If False, perform Welch's
+        t-test, which does not assume equal population variance [2]_.
```

Comment:
```suggestion
        If True (default), perform a standard independent 2 sample test
        that assumes equal population variances [1]_.
        If False, perform Welch's t-test, which does not assume equal
        population variance [2]_.
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.